package.json

```diff
@@ -18,7 +18,9 @@
   "homepage": "https://github.com/meetup/swarm-animation#readme",
   "dependencies": {
     "autoprefixer": "^7.1.1",
-    "bourbon": "^4.2.7"
+    "bourbon": "^4.2.7",
```

Comment:
hm, maybe not? is there anything functionality I would be missing? I can take out